mstm/__init__.py

```diff
@@ -40,9 +40,11 @@
 import os
 import shutil
 import numpy as np
+from scipy import interpolate
+from scipy import integrate
 
 # change to match the filename of your executable
-MSTM_EXE = 'mstm.exe'
+MSTM_EXE = 'mstm_ubuntu.exe'
```

Comment:
I can change it to save the .inp and .out files in the user's path instead, so should I just have it delete those files from the user's path in the end, instead of from the .tmp directory?